assembl/tasks/sudoer.py

```diff
@@ -0,0 +1,307 @@
+import os
+import sys
+from os.path import exists, join
+
+from invoke import task
+from common import (
+    fill_template,
+    is_integration_env,
+    delete_foreign_tasks,
+    get_venv_site_packages,
+    exists as sudo_exists)
+
+# try:
+#     # invoke 0.11
+#     from invoke import ctask as task
+# except ImportError:
+#     from invoke import task
```

Comment:
is this still necessary ?

Reply:
Nope, ctask, which is basically task with `contextualized=True` (means passing the context as first param) became the default in Invoke 0.13 onwards. We're on Invoke 1.2
https://github.com/pyinvoke/invoke/issues/114